new/LIT7030SalamYoungFathers.xml

```diff
@@ -0,0 +1,64 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7030SalamYoungFathers" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ሰላም፡ ለውርዝዋን፡ አበው፡ </title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Salām la-wǝrzǝwān ʾabaw</title>
```

Comment:
Sorry, I missed the issue that led up to the creation of this file, so I will comment here. I wonder if the title is well chosen. Note that the text that is transcribed below in "edition" and that Strelcyn gives begins: "Salutations to you (masc. sing.), the son […] of adult parents". I must confess I am confused by the word ውርዝው – it seems to mean "adult" (as in ready to get married, i.e. still rather young), but then in the next sentence, the salām adds "who begot you in their old age". Well, it's difficult when we don't have the complete text…

Reply:
Your interpretation is convincing. I will change it.

Reply:
I think it means smth a little different (though I agree with the interpretation above, it is a poetic language with transpositions). Wezrew is not only "adult", but also "brave, corageous (man)" . I would  understand ውርዝዋን፡ አበው፡ as "corageous/brave fathers" ("parents" in monastic context, but I would not translate it as "parents"), "who bore you in the old age" (otherwise "young" does not fit "old age", unless a certain figure of speech was intended here, two meanings of "werzew").  

Please style the title as: 

ሰላም፡ ለከ፡ ለውርዝዋን፡ አበው፡ ወልዶሙ፡ 
Salām la-wǝrzǝwān ʾabaw ...
Salutations to you, the son of young parents ... 

This is to show that we quote as the title the first line
(not indicated clearly in the Guidelines, it should actually be, to avoid the confusion around supplied titles and what serves as a title).

Reply:
> ሰላም፡ ለከ፡ ለውርዝዋን፡ አበው፡ ወልዶሙ፡
> Salām la-wǝrzǝwān ʾabaw ...
> Salutations to you, the son of young parents ...

If I understand you correctly, you want me to delete no more than the brackets in the titles.
```
<title xml:lang="gez" xml:id="t1">ሰላም፡ ለከ፡ ለውርዝዋን፡ አበው፡ (...) ወልዶሙ፡ </title>
                <title xml:lang="gez" corresp="#t1" type="normalized">Salām laka la-wǝrzǝwān ʾabaw (...) waldomu</title>
                <title xml:lang="en" corresp="#t1" type="normalized">Salutations to you, the (...) son of young parents</title>
```

I will do it in a new branch.

Reply:
Please do not reduce my comment to moving three dots :) :)

Reply:
Even though, of course you may disagree with my understanding :)

Reply:
I like Jonas translation and changed it in the record, that I have recently merged.